Backpacks.cs

```diff
@@ -666,6 +755,71 @@ public ushort BackpackSize
 
             [JsonProperty("Blacklisted Items (Item Shortnames)")]
             public HashSet<string> BlacklistedItems;
+
+            [JsonProperty(PropertyName = "GUI Button")]
+            public GUIButton GUI = new GUIButton();
+
+            public class GUIButton
+            {
+                [JsonProperty(PropertyName = "Image")]
+                public string Image = "https://i.imgur.com/CyF0QNV.png";
+
+                [JsonProperty(PropertyName = "Background color (RGBA format)")]
+                public string Color = "1 0.96 0.88 0.15";
+
+                [JsonProperty(PropertyName = "GUI Button Position")]
+                public Position GUIButtonPosition = new Position();
+                public class Position
+                {
+                    [JsonProperty(PropertyName = "Anchors")]
+                    public Anchors Anchors = new Anchors();
+
+                    [JsonProperty(PropertyName = "Offsets")]
+                    public Offsets Offsets = new Offsets();
+                }
+
+                public class Anchors
+                {
+                    [JsonProperty(PropertyName = "Anchor Min X")]
+                    public float AnchorMinX = 0.5f;
+
+                    [JsonProperty(PropertyName = "Anchor Min Y")]
+                    public float AnchorMinY = 0.0f;
+
+                    [JsonProperty(PropertyName = "Anchor Max X")]
+                    public float AnchorMaxX = 0.5f;
+
+                    [JsonProperty(PropertyName = "Anchor Max Y")]
+                    public float AnchorMaxY = 0.0f;
+
+                    [JsonIgnore]
+                    public string AnchorsMin => $"{AnchorMinX} {AnchorMinY}";
+
+                    [JsonIgnore]
+                    public string AnchorsMax => $"{AnchorMaxX} {AnchorMaxY}";
+                }
+
+                public class Offsets
+                {
+                    [JsonProperty(PropertyName = "Offset Min X")]
+                    public int OffsetMinX = 185;
+
+                    [JsonProperty(PropertyName = "Offset Min Y")]
+                    public int OffsetMinY = 18;
+
+                    [JsonProperty(PropertyName = "Offset Max X")]
+                    public int OffsetMaxX = 245;
+
+                    [JsonProperty(PropertyName = "Offset Max Y")]
+                    public int OffsetMaxY = 78;
+
+                    [JsonIgnore]
+                    public string OffsetsMin => $"{OffsetMinX} {OffsetMinY}";
+
+                    [JsonIgnore]
+                    public string OffsetsMax => $"{OffsetMaxX} {OffsetMaxY}";
+                }
```

Comment:
My personal preference to reduce the code size here would be to condense the whole position config to just 4 properties (AnchorMin, AnchorMax, OffsetMin, OffsetMax). Most people who can edit it aren't going to need the property separation anyway.